### ISSUE TITLE
Feature request: `isNumeric`

### ISSUE THREAD

**Issue by vorillaz** [plexis-js/plexis#9]
Checks whether the input is a string representation of a numeric value.

**Example usage**

```javascript
import isNumeric from '@plexis/is-numeric';
isNumeric('1000');
// => true

isNumeric('Infinity');
// => true

isNumeric('-100.4');
// => true

isNumeric('1E+2');
// => true

isNumeric('0xFF');
// => true

isNumeric('-');
// => false

isNumeric('one');
// => false
```

**Comment by lambk** [comment]
I got this one

**Comment by Nauthiz-Jera** [comment]
Mind if I work on this issue?

**Comment by vorillaz** [comment]
Closed as it has been implemented #66
